exporters/otlp/test/otlp_metrics_serialization_test.cc

```diff
@@ -140,7 +140,7 @@ TEST(OtlpMetricSerializationTest, Histogram)
 
 TEST(OtlpMetricSerializationTest, ObservableGauge)
 {
-  metrics_sdk::MetricData data = CreateObservableGaugeAggregationData();
+  metrics_sdk::MetricData data = CreateUInt64ObservableGaugeAggregationData();
```

Comment:
this method is creating double type values, rename as double or keep as earlier.

Reply:
thanks, fixed